kong/router.lua

```diff
@@ -1829,13 +1829,15 @@ function _M.new(routes)
   self.select = find_route
   self._set_ngx = _set_ngx
 
+  local server_name = require("ngx.ssl").server_name
```

Comment:
We should be able to move this to the module level, since `require` it in the builder no longer makes sense. (It was done before probably to avoid penalty to HTTP subsystem as it was the main use case).

Reply:
done.